est/004.est

```diff
@@ -0,0 +1,24 @@
+date: 13-8-2016
+author: Mihai Andronache
+method: champions.pert
+scope:
+  1: Database layer changes
```

Comment:
@amihaiemil what do you mean by changes here? Estimation should be done from the project start to the end (at least what we see as end)